src/main/java/core/basesyntax/AppZoo.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AppZoo {
+    public static void main(String[] args) {
+        List<ZooAnimals> zoo = new ArrayList<>();
```

Comment:
Let's create lists for each interface type and call it's method in loop ( swim(), fly() ...